x-pack/plugins/security_solution/server/endpoint/services/workflow_insights/index.test.ts

```diff
@@ -225,16 +226,18 @@ describe('SecurityWorkflowInsightsService', () => {
       await securityWorkflowInsightsService.start({ esClient });
       const insightId = 'some-insight-id';
       const insight = getDefaultInsight();
-      await securityWorkflowInsightsService.update(insightId, insight);
+      const indexName = 'backing-index-name';
+      await securityWorkflowInsightsService.update(insightId, insight, indexName);
 
       // ensure it waits for initialization first
       expect(isInitializedSpy).toHaveBeenCalledTimes(1);
       // updates the doc
       expect(esClient.update).toHaveBeenCalledTimes(1);
       expect(esClient.update).toHaveBeenCalledWith({
-        index: DATA_STREAM_NAME,
+        index: indexName,
```

Comment:
is this one different from DATA_STREAM_NAME that is still being used eg in line 213?

Reply:
The test on line 213 uses the data stream name, but the update method requires the name of the backing index, not the data stream itself.